### DISCUSSION TITLE
Was passed main parameter

### DISCUSSION THREAD

**Discussion by androiddevcoding** [detekt/detekt#3517] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/11345578/110119114-c5ccbc80-7dd4-11eb-88db-ac79ea88c76b.png)
What is the problem?

**Comment by cortinico** [comment, marked as answer]
The problem is that if you invoke with `--baseline` you need to pass the path to the baseline file.
So the invocation will be something like:

```
./detekt-cli-1.15.0/bin/detekt-cli --baseline <path-to-your-baseline.xml-file>
```

**Reply by androiddevcoding** [reply]
![image](https://user-images.githubusercontent.com/11345578/110161934-781d7780-7e07-11eb-85f3-f74077a57c43.png)
OK. What's wrong with that?

**Reply by cortinico** [reply]
`--create-baseline` is a boolean flag.
You need to invoke like:

```
./detekt-cli -i . --create-baseline -b ./baseline.xml
```

Also please refer to this page for the CLI documentation:
https://detekt.github.io/detekt/cli.html

**Reply by androiddevcoding** [reply]
I have looked at the documentation many times.
You need to add examples, this is not very clear. I lost the whole day and only your answer helped me, thx!

**Reply by cortinico** [reply]
I agree :( Docs on CLI could be improved. If you could send us a PR editing this page with the examples you found useful: https://github.com/detekt/detekt/blob/master/docs/pages/gettingstarted/cli.md that would be awesome 🙏
